website/docs/advanced/verification-methods.md

```diff
@@ -23,23 +32,30 @@ const customVerifier = {
       reason: {
         code: 0,
         codeString: "SKIPPED",
-        message: `Document doesn't have version equal to 'https://schema.openattestation.com/2.0/schema.json'`
-      }
+        message: `Document doesn't have version equal to 'https://schema.openattestation.com/2.0/schema.json'`,
+      },
     };
   },
-  test: () => document.version === "https://schema.openattestation.com/2.0/schema.json"
-}
+  test: () => document.version === "https://schema.openattestation.com/2.0/schema.json",
+  // ... below is verify function
```

Comment:
We haven't introduced the concept of verify + the order doesn't matter. Let's remove this sentence

Reply:
ok done.